### DISCUSSION TITLE
280v on adam v2 bootloop on opendingux logo(loading)

### DISCUSSION THREAD

**Discussion by piklz** [eduardofilo/RG350_adam_image#274] [General]
used the migrator before hand 
bootsup then loops on the logo for loading the simple menu i presume

fisrt ideas is sdcard ext not mounting anymore so favs is missing and hangs the simpmenu?

**Comment by eduardofilo** [comment]
In case the external card does not mount, a boot loop would not be possible but you should see the following:

![IMG_20220710_151709](https://user-images.githubusercontent.com/2923728/178146650-029a269c-a474-4c70-9620-1678ad2f1512.jpg)

Which makes me think that your problem is not that the card won't mount, but that there is something about it that SimpleMenu doesn't like. Start better with a blank card, best formatted in FAT32, as exFAT support is still new and untested. Once the system boots as expected and the directory structure is generated on the card, start transferring little by little what you had on the previous card until you find what is causing the problem.
